### ISSUE TITLE
Soll das Einlesen von Rechnungen ZUGFeRD-EXTENDED-Format auch möglich sein?

### ISSUE THREAD

**Issue by dsshad** [LandrixSoftware/XRechnung-for-Delphi#36]
Mit dem neuen Release kam die Option hinzu, Rechnungen im ZUGFeRD-EXTENDED-Format zu erstellen. Ging es ausschließlich um die Erstellung oder müsste auch das Einlesen möglich sein? Ich habe im Beispielprojekt testweise "Gesamtbeispiel-Alles-ciiextended-232.xml" aus dem neuen Ordner "ValidXMLExamples" ausgewählt und auf "Validieren" geklickt. Das Ergebnis war, dass kein Prüfszenario gegriffen hat und die Rechnung abgewiesen werden sollte.

Habe es mit {$DEFINE ZUGFeRD_Support} an und aus probiert.

Jetzt weiß ich nur nicht, ob das Einlesen einfach nicht eingebaut ist oder ob es eigentlich gehen müsste und etwas dementsprechend noch nicht ganz passt.

**Comment by landrix** [comment]
Es geht sowohl ausgeben als auch einlesen 

Die Validierung von ZUGFeRD erfolgt über eine andere Funktion, die ist an dieser Stelle nicht eingebaut, ich bessere das nach

**Comment by landrix** [comment]
erledigt

**Comment by dsshad** [comment]
Ok, habe ich eben ausprobiert. Soll das Ergebnis so in der Form sein?

```
<?xml version="1.0" encoding="UTF-8"?>

<validation filename="Austauschteilesteuer-ciiextended-232.xml" datetime="2024-12-03 16:11:11">
  <xml>
    <info>
      <version>2</version>
      <profile>urn:cen.eu:en16931:2017#conformant#urn:factur-x.eu:1p0:extended</profile>
      <validator version="2.15.0"/>
      <rules>
        <fired>136</fired>
        <failed>0</failed>
      </rules>
      <duration unit="ms">4771</duration>
    </info>
    <summary status="valid"/>
  </xml>
  <summary status="valid"/>
</validation>

```

**Comment by landrix** [comment]
ja, mehr liefert der mustang cmd nicht. Aber ich hab noch was in der Pipeline, ZUGFeRD-Validation mit Schematron, wie bei XRechnung
